docs/find-duplicate-skus/README.md

```diff
@@ -0,0 +1,47 @@
+# Find all Exact Duplicate SKUs
```

Comment:
This library goes for sentence case, so I want to use "Find all duplicate SKUs". (Is the "exact" part important?)

Reply:
I think that 'exact' is redundant when using the word duplicate. As, if it isn't exact, it isn't a duplicate. However, the 'exact' word was a request from PN's peer review in the General Chat so I included it.

Reply:
I switched the title back to "Find duplicate SKUs"; my take on this is that the task documentation (which I expanded a bit on this point) can do the work of clarifying exactly what we consider to be a duplicate. This leaves room for the task to gain options around this in the future (like "Ignore SKU case" or something).